orga/app/components/team/members-list-item.hbs

```diff
@@ -28,13 +28,13 @@
   {{#if this.isEditionMode}}
     <td>
       <div id="selectOrganizationRole">
-        <Ui::SelectInput
+        <PixSelect
           class="table__input"
+          aria-label="{{t 'pages.team-members.actions.select-role.label'}}"
+          placeholder="{{t 'pages.team-members.actions.select-role.label'}}"
           @onChange={{this.setRoleSelection}}
           @options={{this.organizationRoles}}
           @selectedOption={{@membership.organizationRole}}
-          @placeholder="{{t 'pages.team-members.actions.select-role.label'}}"
-          @ariaLabel="{{t 'pages.team-members.actions.select-role.label'}}"
         />
       </div>
     </td>
```

Comment:
**question** : ici on pourrait utiliser `Table::HeaderFilterSelect` directement non ?

Reply:
`Table::HeaderFilterSelect` est  utilisé dans les headers des tableaux `th` ici on est dans le body du tableau `td` donc on peut vraiment le réutilisé tel quel

Reply:
you're right